test/net/sourceforge/kolmafia/session/ResultProcessorTest.java

```diff
@@ -90,4 +95,28 @@ public void gettingCosmicBowlingBallInCombatResetsReturnCombats() {
 
     assertThat("cosmicBowlingBallReturnCombats", isSetTo(-1));
   }
+
+  private static AdventureResult GOBLIN_WATER = ItemPool.get(ItemPool.GOBLIN_WATER);
+
+  @Test
+  public void gettingGoblinWaterFromAquagoblinCompletesGoblinQuest() {
+    QuestDatabase.setQuestProgress(QuestDatabase.Quest.GOBLIN, QuestDatabase.STARTED);
+    MonsterData aqua_g_monster = MonsterDatabase.findMonster("Aquagoblin");
+    MonsterStatusTracker.setNextMonster(aqua_g_monster);
+    KoLCharacter.setPath(AscensionPath.Path.HEAVY_RAINS);
+    ResultProcessor.processResult(true, GOBLIN_WATER);
+    Boolean QuestResult = QuestDatabase.isQuestFinished(QuestDatabase.Quest.GOBLIN);
```

Comment:
Use "boolean questResult"; no need to box the boolean into a Boolean.

Reply:
no need to have a variable when we can just test that in the test, like the second test.